spec/ident_http_spec.rb

```diff
@@ -14,11 +14,21 @@
                    excluded_uris.include? check
```

Comment:
as per the private repo comment:

If the number of specs is going to increase and diversify in type might be worth starting with a folder structure that matches the project structure.

Ex:
spec/checks/http
spec/checks/*
spec/lib/*

will make it easier to manage and easier to navigate.

Reply:
agreed, love this idea so lets do it.